kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -251,7 +251,7 @@ class ResidualBasedBlockBuilderAndSolver
 
 
             //#pragma omp parallel for firstprivate(nconditions, LHS_Contribution, RHS_Contribution, EquationId ) schedule(dynamic, 1024)
-            #pragma omp parallel for  schedule(guided, 512)
+            #pragma omp for  schedule(guided, 512)

```

Comment:
May I ask a (may be naive) question. Why you are using a fix optimum chunk size? and how come the 512?